.circleci/config.yml

```diff
@@ -358,93 +367,93 @@ jobs:
           environment:
             GITHUB_TOKEN: $GITHUB_TOKEN
           command: |
+            mkdir binaries
+            find artifacts -type f -exec cp -t binaries {} +
```

Comment:
I could just flatten artifacts by `mv`ing the bins out of the target dirs here, rather than copying to a new dir.